src/csharp/Microsoft.Spark/Sql/PicklingUdfWrapper.cs

```diff
@@ -21,7 +28,7 @@ internal PicklingUdfWrapper(Func<TResult> func)
             _func = func;
         }
 
-        internal object Execute(int splitIndex, object[] input, int[] argOffsets)
+        internal object Execute(int _, object[] input, int[] argOffsets)
```

Comment:
Can we either have a comment about this param or revert the change?

Reply:
readded